libnest/src/config.rs

```diff
@@ -59,10 +63,26 @@ impl Config {
     /// ```
     #[inline]
     pub fn new() -> Config {
-        Config {
-            cache: PathBuf::from("/var/lib/nest/cache/"),
-            download_path: PathBuf::from("/var/lib/nest/download/"),
-            repositories: Vec::new(),
+        Config::default()
+    }
+
+    /// Loads a configuration from a TOML file
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # extern crate libnest;
+    /// use libnest::config::Config;
+    ///
+    /// let mut config = Config::new();
+    /// config.load_conf("Config.toml");
+    /// ```
+    pub fn load_conf(&mut self, path: &str) {
+        match ConfigParser::new(path) {
+            Ok(conf_parser) => {
+                conf_parser.load_to_config(self);
+            }
+            Err(e) => eprintln!("Error when parsing configuration file '{}': {}", path, e),
```

Comment:
You're doing it the wrong way.

You're writting a library, this implies that you **can't take any final decisions**, which means you can't handle errors at this level, you must report them to the caller so they can be displayed correctly.

Remember, `libnest` is only a backend to a potential GUI front-end. Errors on the terminal will be useless to the user in such context. It's something that you should constantly have in mind.